Formula/neomutt.rb

```diff
@@ -0,0 +1,72 @@
+class Neomutt < Formula
+  desc "Teaching an Old Dog New Tricks"
+  homepage "https://www.neomutt.org/"
+  url "https://github.com/neomutt/neomutt.git", :tag => "neomutt-20171215", :revision => "ae61285170533b4be544e738cdd2eedefd1856ff"
+  head "https://github.com/neomutt/neomutt.git", :branch => "master"
+
+  option "with-lmdb", "Build with lmdb support"
```

Comment:
The formula shouldn't have any options. Pick what you think the best set of dependencies is.

Reply:
@jindraj 
I suggest building-in **both** tokyocabinet (traditional default) and lmdb (fast modern alternative)
We definitely want to offer Notmuch support.
We don't need lua or slang options for the general user.